src/lib/server/drizzle/table/card.ts

```diff
@@ -0,0 +1,34 @@
+import { mysqlEnum, mysqlTable, varchar } from 'drizzle-orm/mysql-core';
+import { userTable } from './user';
+
+const Categories = {
+  first: 'FIRST',
+  second: 'SECOND',
+  third: 'THIRD',
+  fourth: 'FOURTH',
+  fifth: 'FIFTH',
+  sixth: 'SIXTH',
+  seventh: 'SEVENTH',
+  done: 'DONE',
+};
```

Comment:
```suggestion
export const Categories = {
  first: 'FIRST',
  second: 'SECOND',
  third: 'THIRD',
  fourth: 'FOURTH',
  fifth: 'FIFTH',
  sixth: 'SIXTH',
  seventh: 'SEVENTH',
  done: 'DONE',
} as const;
```

Mettre dans un autre fichier